include/deal.II/grid/reference_cell.h

```diff
@@ -45,6 +45,38 @@ namespace ReferenceCell
     Invalid = static_cast<std::uint8_t>(-1)
   };
 
+  /**
+   * Convert the given reference cell type to a string.
+   */
+  inline std::string
+  type_to_string(const Type &type)
+  {
+    switch (type)
+      {
+        case Type::Vertex:
+          return "Vertex";
+        case Type::Line:
+          return "Line";
+        case Type::Tri:
+          return "Tri";
+        case Type::Quad:
+          return "Quad";
+        case Type::Tet:
+          return "Tet";
+        case Type::Pyramid:
+          return "Pyramid";
+        case Type::Wedge:
+          return "Wedge";
+        case Type::Hex:
+          return "Hex";
+        case Type::Invalid:
+        default:
```

Comment:
Since you fall through from `Invalid` to `default`, you don't actually need the last `case` statement.

Reply:
Let me fix this in another way - we might want to convert `Invalid` to a string after all.